src/effects/effectsmanager.cpp

```diff
@@ -307,8 +354,18 @@ void EffectsManager::saveEffectsXml() {
         quickEffectChainPresets.insert(qeIt.key(), pPreset);
     }
 
+    QHash<QString, EffectChainPresetPointer> quickStemEffectChainPresets;
+    quickStemEffectChainPresets.reserve(m_quickStemEffectChains.size());
+    QHashIterator<QString, QuickEffectChainPointer> sqeIt(m_quickStemEffectChains);
```

Comment:
```suggestion
    QHashIterator<QString, QuickEffectChainPointer> qseIt(m_quickStemEffectChains);
```
I guess the meaning should be QuickStemEffectIterator?